### PR TITLE
[Issue #154] feat:wallet ui

### PR DESCRIPTION
**Pull request by johnkuney**
Adding some UI for the wallets page. Using mock data for now. Basically have 
-wallet name 
- network
- Current balance (wasnt sure if we wanted this, or more like on the dashboard where its total received?)
- payments
- Button list, and each one links to the specific button page
- edit button  
- create new wallet button

### CODE REVIEW

**Review by Klakurka — CHANGES_REQUESTED** [review]
We're going to break a crypto convention here intentionally and allow them to have any number of currencies within a wallet (in the same way you can in a physical fiat wallet).

So instead of showing the 'eCash Network' text, let's show the currency icons for whatever networks they have payments on in the wallet on the buttons. So they won't need to "select" a currency when making a new wallet. The button's addresses contained within the wallet dictate what networks we show there. If they're on both XEC and BCH, just put both icons next to each other I guess unless you have a better idea.

**Comment by Klakurka** [comment]
Let's also add an actual 'Default Wallet' marker that can set in the wallet edit dialog.

**Comment by johnkuney** [comment]
For sure, added network icons, and separated out the balances

File: pages/wallets/index.tsx [hunk 1]
```diff
@@ -34,14 +36,53 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
...
   )
 }
+
+class ProtectedPage extends React.Component<WalletsProps, WalletsState> {
+  constructor (props: WalletsProps) {
+    super(props)
+    this.state = {
+      wallets: [
+        {
+          name: 'Default Wallet',
```
Comment (Klakurka) [inline pages/wallets/index.tsx:61]: We should have an actual field here for 'default_wallet' or whatever which is a true/false field.

Can we also have some sort of indicator somewhere that tells people that this is the default wallet (even if they change the wallet name)?

**Review by Klakurka — CHANGES_REQUESTED** [review]
Nit but I think the vertical bar to the left of the icons should be the same font colour as the text.

Everything else looks good.

**Comment by Klakurka** [comment]
I see that the vertical bar shows up in a few places. Can we go for something really subtle? The light theme is fine I think but the dark theme is a blue colour which I think should be changed to a dark grey or something so that it's not what the eyes are drawn to.

**Comment by johnkuney** [comment]
Cool, added. Added edit wallet form too

**Review by Klakurka — APPROVED** [review]
Looks good.

I overlooked something though that we may need to sort out at some point...

We might need to do some brainstorming about how we want to approach this but basically we need to have a default for each currency I think.

Let's cross that bridge later as we run into actual issues.

**Pull request merged**